renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -1456,6 +1456,10 @@ SERIALISE_VK_HANDLES();
   PNEXT_STRUCT(VK_STRUCTURE_TYPE_SUBPASS_FRAGMENT_DENSITY_MAP_OFFSET_END_INFO_QCOM,                    \
                VkSubpassFragmentDensityMapOffsetEndInfoQCOM)                                           \
                                                                                                        \
+  /* VK_EXT_physical_device_drm */                                                                     \
+  PNEXT_STRUCT(VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DRM_PROPERTIES_EXT,                                   \
+               VkPhysicalDeviceDrmPropertiesEXT)                                                       \
```

Comment:
nit: these extensions are listed in alphabetical order (and in EXT/KHR tag groups) could you move this up?